### DISCUSSION TITLE
designable 表单项数据源什么之后支持远程数据呢

### DISCUSSION THREAD

**Discussion by aidening** [alibaba/designable#99] [General]
现在的 `designable form item`  的数据源是静态数据源，可以支持配置一个接口或者选择一个接口获取数据源吗？

例如：下拉列表的 options 现在是一项一项添加，可以直接配置接口吗？

![image](https://user-images.githubusercontent.com/45580373/137111051-5642d841-aa13-416f-a4c6-d061cfc050d2.png)

**Comment by janryWang** [comment]
响应器里自己写逻辑
